### PR TITLE
update spanish localization

### PR DESCRIPTION
**Pull request by Victor-M-Giraldo**

### CODE REVIEW

File: client/src/locales/es.ts [hunk 1]
```diff
@@ -42,7 +55,7 @@ export default {
 			name: "OpenTogetherTube",
 			text1: "Es una plataforma de sincronizacion de videos en tiempo real.\nEs fácil para usar y no tienes que inscribirte. Simplemente crea un sala, agrega videos y\ninvita a tus amigos. BOOM! Estás listo para disfrutar de videos con tus amigos hasta las 3 de la mañana",
 			text2: "El TogetherTube original fue amado por su interfaz simple,\ny lo fácil que fue empezar a ver vídeos de inmediato.\nOpenTogetherTube pretende ser igual de fácil de usar y luego mejoralo\npara hacerlo aún mejor.",
-			text3: "Actualmente, puedes ver videos en línea con tus amigos de Youtube, Vimeo, enlaces directos a videos .mp4 y",
+			text3: "Actualmente, puedes ver videos en línea con tus amigos de Youtube, Vimeo, Dailymotion, enlaces directos a videos .mp4 y",
```
Comment (dyc3) [inline client/src/locales/es.ts:58]: Dailymotion support was disabled in #1670

File: client/src/locales/es.ts [hunk 2]
```diff
@@ -185,6 +199,7 @@ export default {
 		"platforms": {
 			"youtube-videos": "Vídeos de Youtube: {url}",
 			"vimeo-videos": "Vídeos de Vimeo: {url}",
+			"dailymotion-videos": "Vídeos de Dailymotion: {url}",
```
Comment (dyc3) [inline client/src/locales/es.ts:202]: Dailymotion support was disabled in #1670

**Review by dyc3 — CHANGES_REQUESTED** [review]
Mostly a rubber stamp.

File: client/src/locales/es.ts (hunk 1 above)
Comment (Victor-M-Giraldo) [reply client/src/locales/es.ts:58]: Oops, yea. I pasted this over from a branch that was behind.

**Review by dyc3 — APPROVED** [review]

**Pull request merged**